openmetadata-docs/content/v1.6.x-SNAPSHOT/menu.md

```diff
@@ -825,8 +817,15 @@ site_menu:
     isCollateOnly: true
   - category: How-to Guides / Data Quality and Profiler / Data Quality / How to Write and Deploy No-Code Test Cases
     url: /how-to-guides/data-quality-observability/quality/test
-  - category: How-to Guides / Data Quality and Observability / Data Quality / Configure Data Quality
-    url: /how-to-guides/data-quality-observability/quality/configure
```

Comment:
Looks like this got replaced instead, can you take a look? Thanks

Reply:
yes this got replaced and again added in the section